client/packages/common/src/hooks/useDeleteConfirmation/useDeleteConfirmation.ts

```diff
@@ -32,10 +32,13 @@ export const useDeleteConfirmation = <T>({
   } = messages;
   const t = useTranslation('common');
   const { success, info } = useNotification();
-  const cannotDeleteSnack = info(
-    cantDelete || t('messages.cant-delete-generic')
-  );
 
+  const cannotDeleteSnack = (errorMessage?: string) => {
```

Comment:
following up on this one. All of the calls return the server error, which will be an unfriendly looking message, apart from the SidePanel implementations which aren't even catching errors!

- useStocktakeDeleteSelected
- useStocktakeDeleteSelectedLines
- Stocktake/DetailView/SidePanel isn't catching errors, probably should check this one
- useInboundDeleteRows
- InboundShipment/DetailView/SidePanel isn't catching errors either
- useOuboundShipmentDeleteRows
- useOuboundShipmentDeleteSelectedLines
- OutboundShipment/DetailView/SidePanel isn't catching errors either
- usePrescriptionDeleteRows
- usePrescriptionDeleteSelectedLines
- Prescriptions/DetailView/SidePanel isn't catching errors either
- useDeleteSelectedRequisitions
- useDeleteRequestLines
- RequestRequisition/DetailView/SidePanel isn't catching errors either

Thoughts about how to handle those? It would be worth tying to get an error and see what sort of message we show. It might be nothing, but I'm worried that we'll be replacing a generic message with a long string of meaningless text

Reply:
Hey Mark, 

I'm wondering about leaving the toast as it was previously with just the generic message (the original issue was due to receiving a success snack rather than an error snack), and mapping more useful errors if required in another issue.

What do you think of that?

Happy to work through the use cases but wondering if it is worthwhile right now.

Reply:
yeah, probably wise. adding in the error translation would take this issue off course. I really liked the error parsing you had, though if we want to unpack errors like that we should probably demand that the API has structured errors which are more easily translatable.